### PR TITLE
Updates to chapters 1 & 2 - mostly factual, a little style

### PR DESCRIPTION
**Pull request by mpdaugherty**
I went through chapters 1 and 2 looking for factual updates (e.g. version numbers, etc.)

In Chapter 2, I also found a few places where the transition from SVN to Git required rephrasing a few sentences.

The other large change in Chapter 2 had to do with SQLite; since Django now requires Python 2.5 or higher, I took out instructions on how to get SQLite working with Python 2.4.

### CODE REVIEW

**Comment by mpdaugherty** [comment]
@jacobian Do we have a style guideline for the .rst files?  I automatically removed whitespace at the end of the lines and wrapped paragraphs at 80 characters, but I'm afraid that this may make the diffs hard to read.  Let me know if it's appropriate or not.  Thanks!

**Comment by jacobian** [comment]
@mpdaugherty generally, I do exactly what you said - reformat and re-wrap as needed. It does make diffs slightly harder to read, but it makes the source easier to read so that's a tradeoff I'm OK with.

Thanks for the changes, I'll try to review and merge or let you know if there's anything that needs fixing.

File: chapter01.rst [hunk 1]
```diff
@@ -335,20 +335,20 @@ print by Apress.
...
-1.1, it will still work for 1.2, 1.3, 1.9 and any other version number that
+1.4, it will still work for 1.5, 1.6, 1.9 and any other version number that
 starts with "1."
 
 Once Django hits 2.0, though, your applications might need to be rewritten --
 but version 2.0 is a long way away. As a point of reference, it took more than
 three years to release version 1.0. (This is very similar to the compatibility
 policy taken by the Python language itself: code that was written for Python
-2.0 works on Python 2.6, but not necessarily with Python 3.0.)
+2.0 works on Python 2.7, but not necessarily with Python 3.0.)
```
Comment (jacobian) [inline chapter01.rst]: This actually doesn't match the our current thinking about Django's release policy any more -- the plan is to always be backwards-compatible, at least as much as possible. So let's nix this whole paragraph.

File: chapter02.rst [hunk 2]
```diff
@@ -36,14 +36,12 @@ Python 2.5, so using a later version of Python keeps your options open.
...
+    supported it experimentally. Python 3.0 introduced a substantial number of
     backwards-incompatible changes to the language itself, and, as a result,
-    we expect most major Python libraries and frameworks, including Django,
-    will take a few years to catch up.
-
-    If you're new to Python and are wondering whether to learn Python 2.x or
-    Python 3.x, our advice is to stick with Python 2.x.
+    many major Python libraries and frameworks, including Django, had not yet
+    caught up.  Python 3.0 support is expected in Django version 1.5 at the
+    earliest.
```
Comment (jacobian) [inline chapter02.rst]: "at the earliest" is actually wrong - we made it happen, and 1.5 will support 3.2 and above. This should reflect that.

File: chapter01.rst (hunk 1 above)
Comment (mpdaugherty) [inline chapter01.rst]: Great; will do

File: chapter02.rst (hunk 2 above)
Comment (mpdaugherty) [inline chapter02.rst]: Ah, I wasn't sure and didn't find a good source while googling.  Will update.

**Comment by jacobian** [comment]
Awesome, thanks!

**Pull request merged**